userguide/config.toml

```diff
@@ -1,21 +1,22 @@
 baseURL = "/"
 title = "Docsy Example"
 
-enableRobotsTXT = true
-
-# Hugo allows theme composition (and inheritance). The precedence is from left to right.
-theme = ["docsy"]
-
-# Will give values to .Lastmod etc.
-enableGitInfo = true
-
 # Language settings
 contentDir = "content/en"
 defaultContentLanguage = "en"
 defaultContentLanguageInSubdir = false
 # Useful when translating.
 enableMissingTranslationPlaceholders = true
 
+enableRobotsTXT = true
+
+# set docsy as theme, either locally or as hugo module
```

Comment:
If we decide to keep this change, then maybe:

```toml
theme = ["docsy"] # Use this repo as the theme
# theme = ["github.com/google/docsy", "github.com/google/docsy/dependencies"] # Use docsy Hugo modules
```

Reply:
If me follow my proposal given in my [comment](https://github.com/google/docsy/pull/871#issuecomment-1036220370) below, then we should have the `[module]` declaration only, superseding the traditional `theme = "..."` approach.

Reply:
Put in the theme assignment as found in HEAD of repo (`master` branch), as requested.